mocha_test/mapValues.js

```diff
@@ -0,0 +1,92 @@
+var async = require('../lib');
+var expect = require('chai').expect;
+var assert = require('assert');
+
+describe('mapValues', function () {
+    var obj = {a: 1, b: 2, c: 3};
```

Comment:
`{0: <val>, 1: <val>, 2: <val>}`

I would also be fine with leaving this behaviour implied